src/components/subscriptions/SubscriptionManagementPage.jsx

```diff
@@ -14,10 +14,12 @@ import StatusAlert from '../StatusAlert';
 import './styles/SubscriptionManagementPage.scss';
 import { TAB_PENDING_USERS } from './constants';
 import AddUsersButton from './AddUsersButton';
-import { configuration } from '../../config';
 
 const PAGE_TITLE = 'Subscription Management';
-const { CUSTOMER_SUPPORT_EMAIL } = configuration;
+
+// get current url and remove the 'subscription'
+const CURRENT_URL = window.location.pathname;
```

Comment:
I want to say you can do something like `window.location.origin` to get the `base_url`? Or there's probably some similar property so we don't have to pull it out so violently